user_docs/en/userGuide.t2t

```diff
@@ -1684,6 +1684,18 @@ You can move from braille table to braille table in the list by using the arrow
 Note that this option is only useful if your braille display has a Perkins-style keyboard and this feature is supported by the braille display driver.
 If input is not supported on a display which does have a braille keyboard, this will be noted in the [Supported Braille Displays #SupportedBrailleDisplays] section.
 
+%kc:setting
+==== Braille mode ====[BrailleMode]
+Key: ``NVDA+alt+t``
+
+This option allows you to select between the available braille modes.
+
+Currently, two braille modes are supported, "follow cursors" and "display speech output".
+
+When follow cursors is selected, the braille display will follow either the system focus/caret or the navigator object/review cursor, depending on what braille is tethered to.
+
+When display speech output is selected, the braille display will show what NVDA would have spoken if speech mode is set to "talk"

```

Comment:
Writing this way may let think that using "display speech" would prevent NVDA from talking (e.g. force speech mode to off) or that people should use another speech mode than "talk" when using this feature.

Also "is" -> "was"; please confirm English grammar (cc @seanbudd or @XLTechie)
```suggestion
When display speech output is selected, the braille display will show what NVDA speaks, or would have spoken if speech mode was set to "talk"
```